Seasonizer/AccessoryListViewController.swift

```diff
@@ -15,11 +15,15 @@ class AccessoryListViewController: UITableViewController {
     
     var accessories: [Accessory]?
     var selectedAccessory: Accessory?
-    
-    
     // MARK: User Interaction
     
     // TODO: Implement `prepareForSegue(_:sender:)` to set `selectedAccessory` when the "selectedAccessory" Segue is performed.
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        guard let indexPath = tableView.indexPathForSelectedRow else {
```

Comment:
Du solltest hier immer den `segue.identifier` abfragen, z.B. in einem `switch`, selbst wenn du nur eine Segue hast. Falls du später noch mehr Segues hinzufügst gibt's sonst komische Fehler.